haskell/private/versions.bzl

```diff
@@ -38,23 +38,81 @@ def _parse_bazel_version(bazel_version):
     """
     return [int(_parse_version_chunk(x)) for x in bazel_version.split(".")]
 
-def check_version(actual_version):
-    if type(actual_version) != "string" or len(actual_version) < 5:
-        return  # Unexpected format
+def _is_at_least(threshold, version):
+    """Check that a version is higher or equals to a threshold.
+    Args:
+      threshold: the minimum version string
+      version: the version string to be compared to the threshold
+    Returns:
+      True if version >= threshold.
+    """
+    # Vendored from https://github.com/bazelbuild/bazel-skylib/blob/e30197f3799eb038fbed424e365573f493d52fa5/lib/versions.bzl
```

Comment:
These are vendored because Bazel warning check occurs in `haskell/repositories.bzl` which is the place that imports Skylib in the first place. So, we can't depend on it here.